### PR TITLE
[HOU-2021] Added PayPal sponsor links, added location page, added COVID-19 state…

### PR DESCRIPTION
**Pull request by skryukova**
…ment

*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: content/events/2021-houston/location.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
 +++
 Title = "Location"
 Type = "event"
-Description = "Location for devopsdays Houston 2021"
+Description = "Location for devopsdays Houston 2019"
```
Comment (MikeRosTX) [inline content/events/2021-houston/location.md:4]: Why was this changed to 2019? Just curious....

Comment (skryukova) [reply content/events/2021-houston/location.md:4]: Thanks for catching this, this is copy and paste error. Will fix it in a minute.
